samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -0,0 +1,197 @@
+
+User interface
+**************
+
+Buttons:
+
+ Can be used to input the out-of-band (OOB) authentication value during provisioning. All buttons have the same functionality during this procedure.
+
+Button 1:
+
+ On press and release, Button 1 will publish a Gen. OnOff Set message using the configured publication parameters of its model instance and toggles the LED on/off on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+
+ On press and hold, Button 1 will publishe a Gen. Move Set message using the configured publication parameters of its model instance and will continuously dim the LED lightness state on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+   On release, the button client publish another Gen. Move Set message with the ``delta`` parameter set to 0, making the continuous level change from the previous message stop.
+
+LEDs:
+
+ Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
+
+.. note::
+   :ref:`zephyr:thingy53_nrf5340` supports only one RGB LED.
+   Each RGB LED channel is used as separate LED.
+
+Configuration
+*************
+
+|config|
+
+|nrf5340_mesh_sample_note|
+
+...
+
+Source file setup
+=================
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* A file for handling mesh models, :file:`model_handler.c`.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_dimmer`
+
+.. include:: /includes/build_and_run_ns.txt
+
+.. _bluetooth_mesh_light_dimmer_testing:
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it by using a smartphone with `nRF Mesh mobile app`_ installed.
+Testing consists of provisioning the device and configuring it for communication with the mesh models.
+
+Provisioning the device
+-----------------------
+
+.. |device name| replace:: :guilabel:`Mesh Light Dimmer`
+
+.. include:: /includes/mesh_device_provisioning.txt
+
+Configuring models
+------------------
+
+See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.
+
+Configure the Generic Level Client model on the first element on the **Mesh Light Dimmer** node:
+
+* Bind the model to **Application Key 1**.
+* Configure the publication to any group.
+
+Configure the first element on the target **Mesh Light Fixture** node:
+
+* Bind the following models to **Application Key 1**: Generic Level Server and Generic OnOff Server.
+* Subscribe both models to the same group as set for publication for the **Mesh Light Dimmer**.
+
+You should now be able to perform the following actions:
+
+* Press and release **Button 1** to see the LED on the target Mesh Light Fixture toggle on and off.
+* Press and hold **Button 1** to see the LED on the target Mesh Light Fixture light level slowly decrease or increase.
```

Comment:
```suggestion
* Press and hold **Button 1** to see the light level of the LED on the target Mesh Light Fixture device slowly decrease or increase.
```

Reply:
Should we use a  ref to the "Mesh Light Fixture" here?

Reply:
As the first mention in this sub-section, or in this particular sentence you mean? @mia-ko

Reply:
I don't think you need reference here.
We are simply referring to the device name visible in the mobile app.
At least that is what I am assuming here.